README.md

```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
 
 Description
 ------------
-With the new VPNService of Android API level 14+ (Ice Cream Sandwich) it is possible to create a VPN service that does not need root access. This project is a port of OpenVPN.
+In Android 4.0 (Ice Cream Sandwich) a VPN service can be made without root access, using the VPNService of API level 14+. \
+This project is a port of OpenVPN.
 
 <a href="https://f-droid.org/repository/browse/?fdid=de.blinkt.openvpn" target="_blank">
 <img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=de.blinkt.openvpn" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
 Developing
----------------
-If you want to develop on ics-openvpn please read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or emailing me. 
+----------
+Read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or e-mailing. 
 
-Also please note that before contributing to the project that I would like to retain my ability to relicense the project for different third parties and therefore probably need a contributer's agreement from any contributing party. To get started, [sign the Contributor License Agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn).
+To retain my ability to re-license the project for different third-parties I probably need a [contributor agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn) from any contributing party.
```

Comment:
fixes a mistake but also changes the tone and removes an explaination.

Reply:
I introduced a mistake too in forgetting "contributor **license** agreement".
Can "probably" be removed? It reads like "pending research" to me.
The tone I interpret as defensive, as opposed to welcoming changes.
"The project" is repeated. I don't know what is lost other than length from a long-winded way to say (I thought) the same thing.